engines/agi/agi.cpp

```diff
@@ -418,13 +418,22 @@ AgiEngine::AgiEngine(OSystem *syst, const AGIGameDescription *gameDesc) : AgiBas
 
 	_game.mouseEnabled = true;
 	_game.mouseHidden = false;
+	_game.predictiveDlgOnMouseClick = true;
+
 	// don't check for Amiga, Amiga doesn't allow disabling mouse support. It's mandatory.
 	if (!ConfMan.getBool("mousesupport")) {
 		// we effectively disable the mouse for games, that explicitly do not want mouse support to be enabled
 		_game.mouseEnabled = false;
 		_game.mouseHidden = true;
 	}
 
+	// Since this option is a late addition to the engine, assume its value to be true if it's not set in the config file
+	// This is done for consistency with the previous behavior (before making this feature optional)
```

Comment:
Thanks
I do agree that this feature is annoying for desktop platforms, and this behavior goes back to at least 2006 with commit 1114480ece2c96b694ce0860c9f7af5fecf15ffa (but this behavior was introduced before that commit).

I believe that it's time to turn this feature off by default, since its usage is limited nowadays, and it's annoying for non-touch platforms.

Thoughts?

Reply:
With the recent commits, the feature is turned off by default. 

This does mean that for games already added, the behavior will become disabled (while previously it was enabled), and the user, if they choose to, will now have to enable it from the GUI.